.github/workflows/release_bundle.yml

```diff
@@ -36,30 +36,67 @@ on:
         required: true
 
 jobs:
+  build-python-matrix:
+    name: Audit Version and Build Python Release Matrix
+    runs-on: ubuntu-latest
+    outputs:
+      python_versions: ${{ steps.build-list.outputs.versions }}
+
+    steps: 
+      - name: "Audit Version And Parse Into Parts"
+        id: semver
+        uses: dbt-labs/actions/parse-semver@v1.1.0
+        with:
+          version: ${{ inputs.version_number }}
+      
+      - name: "Set Python Versions"
+        # 1.0 to 1.3 support python 3.7 to 3.10 
+        # 1.4 added support for python 3.11
+        # 1.6 drops support for python 3.7
+        # Note that Python 3.8 is manually added to the matrix so not included here
+        id: build-list
+        run: |
+          if [[ ${{ steps.semver.outputs.minor }} < 4 ]]
+          then
+            py_versions='["3.7", "3.9", "3.10"]'
```

Comment:
We're dropping support for py37. Do we just need this here until the end of the month and then we'll remove it? Or can we remove it now?

Reply:
We're only dropping support in 1.6 on.  Not old versions.  We still to be able to build bundles for those versions.

Reply:
Ideally we should have this in a config file mapping dbt to python versions but I think this is good enough for now. 

nit: Is it worth it to check that major version == 1? (i.e. if we try to generate a bundle for 2.0 or 0.9 this should fail)

Reply:
Why drop 3.8 but not 3.7 though is my question 🤔

Reply:
@VersusFacit there's a comment right after the job name defining what version needs what version of python.  It ends with a note explaining why 3.8 is not included in this matrix.  

We build for 3.8 linux in the first job and then add it for windows specifically in the second job.

Reply:
Oh perfect. Still getting antiquated with all these 😅